pkg/client/metrics/prometheus/prometheus.go

```diff
@@ -27,26 +27,22 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-const restClientSubsystem = "rest_client"
-
 var (
 	// requestLatency is a Prometheus Summary metric type partitioned by
 	// "verb" and "url" labels. It is used for the rest client latency metrics.
 	requestLatency = prometheus.NewSummaryVec(
```

Comment:
As we can't aggregate quantiles, wouldn't it be a lot more helpful to use a histogram here?